rpc.go

```diff
@@ -94,12 +93,12 @@ func (r rpc) Channel(service string) transport.Channel {
 		// we can eventually write an outbound that load balances between
 		// known outbounds for a service.
 		return transport.Channel{
-			Outbound: out,
+			Outbound: request.ValidatorOutbound{out},
```

Comment:
Will figure out a way to do this

Created https://github.com/yarpc/yarpc-go/issues/109